airbyte-integrations/connectors/source-google-ads/source_google_ads/spec.json

```diff
@@ -39,13 +39,18 @@
             "title": "Refresh Token",
             "description": "Refresh token generated using developer_token, oauth_client_id, and oauth_client_secret. More instruction on how to find this value in our <a href=\"https://docs.airbyte.io/integrations/sources/google-adwords#setup-guide\">docs</a>",
             "airbyte_secret": true
+          },
+          "login_customer_id": {
```

Comment:
this should be a separate field outside of `credentials` since that is an autogenerated json which will likely not contain this information

Reply:
Done